app/assets/javascripts/application.js

```diff
@@ -41,6 +41,15 @@ $(function() {
     // Paginate link sends AJAX request to controller, which renders new page
     // in JS response.  These callbacks execute at that point and replaces
     // the prior pagination page (DOM element) with the new page.
+    var Users = {
+      setup: function () {
+        $('body').on('ajax:success', '#users_list a[data-remote]', function (e, data) {
+          $('#users_list').html(data);
+        });
+      }
+    };
+    Users.setup();
```

Comment:
The idea is that after some refactoring the Users module could go in its own file (users.js by convention) to reduce merge conflicts (since all of the JavaScript files are combined and compressed anyways.) Then the namespacing comes in handy to reduce global variables. By using a setup method, the control logic is separated from the model logic.